File: app/src/main/java/wangdaye/com/geometricweather/main/ui/adapter/daily/model/DailyAirQuality.java
package wangdaye.com.geometricweather.main.ui.adapter.daily.model;

import wangdaye.com.geometricweather.basic.model.weather.AirQuality;
import wangdaye.com.geometricweather.main.ui.adapter.daily.DailyWeatherAdapter;

public class DailyAirQuality implements DailyWeatherAdapter.ViewModel {

    private AirQuality airQuality;

    public DailyAirQuality(AirQuality airQuality) {
        this.airQuality = airQuality;
    }

    public AirQuality getAirQuality() {
        return airQuality;
    }

    public void setAirQuality(AirQuality airQuality) {
        this.airQuality = airQuality;
    }

    public static boolean isCode(int code) {
        return code == 5;
    }

    @Override
    public int getCode() {
        return 5;
    }
}

File: app/src/main/java/org/breezyweather/daily/adapter/model/DailyAirQuality.kt
package org.breezyweather.daily.adapter.model

import org.breezyweather.common.basic.models.weather.AirQuality
import org.breezyweather.daily.adapter.DailyWeatherAdapter

class DailyAirQuality(
    val airQuality: AirQuality
) : DailyWeatherAdapter.ViewModel {
    override val code = 5

    companion object {
        fun isCode(code: Int) = (code == 5)
    }
}
